README.md

```diff
@@ -13,6 +13,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [Great Indian Developer Summit](http://www.developermarch.com/developersummit/index.html) | 24-27 Apr | Bengaluru | 	With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem. |
 | [GopherCon India](http://www.gophercon.in/) | 09-10 Mar | Sheraton Grand, Sangamvadi, Pune | Go conference in India. Go is an open source project developed by a team at Google. |
 | [FOSSMeet '18](http://fossmeet.in/)| 17-18 Feb | NIT, Calicut |FOSSMeet is an annual event on Free and Open source software, conducted at NIT, Calicut. |
+| [Gnunify 2018](http://gnunify.in/) | 16-17 Feb | SICSR, Pune | GNU/Linux and open source technologies and related. |
```

Comment:
Sorry for not adding these comments earlier -- can you update the conference name to `GNUify`? The website says so and it makes more sense this way too. ;)

Reply:
For the description - Conference related to GNU/Linux and open source technologies.